frontend/src/components/worksheets/BundleDetail/SideBar.jsx

```diff
@@ -65,82 +62,181 @@ class SideBar extends React.Component<
+                <div>
+                    <ConfigLabel label="Owner:" inline={true}/>
+                    <div className={classes.dataText}>
+                        { bundleInfo.owner.user_name }
+                    </div>
+                </div>
+                <div>
+                    <ConfigLabel label="Created:" inline={true}/>
+                    <div className={classes.dataText}>
+                        {renderFormat(metadata.created, metadataType.created)}
+                    </div>
+                </div>
+                {metadata.data_size 
+                    ?   <div>
+                            <ConfigLabel label="Size:" inline={true}/>
+                            <div className={classes.dataText}>                    
+                                {renderFormat(metadata.data_size, metadataType.data_size) }
+                            </div>
                         </div>
-                    )
+                    :   null
+                }
+                {/** ----------------------------------------------------------------------------------------------- */}
+                <ConfigLabel label="Dependencies:" />
+                { isRunBundle && <Dependency bundleInfo={ bundleInfo }/>
+                    ? <Dependency bundleInfo={ bundleInfo }/> 
+                    : <div>None</div>}
+                {/** ----------------------------------------------------------------------------------------------- */}
+                <ConfigLabel label="Attached to these Worksheets:" />
+                {
+                    bundleInfo.host_worksheets.length > 0
+                    ?   bundleInfo.host_worksheets.map((worksheet) =>
+                            <div
+                                key={ worksheet.uuid }
+                            >
+                                <a
+                                    href={ `/worksheets/${ worksheet.uuid }`}
+                                    className={ classes.uuidLink }
+                                    target="_blank"
+                                >
+                                    { worksheet.name }
+                                </a>
+                            </div>
+                        )
+                    :   <div>None</div>
                 }
                 {/** ----------------------------------------------------------------------------------------------- */}
-                <div className={classes.spacer}/>
                 <div>
-                    <Typography variant="subtitle1">Permissions</Typography>
-                    <PermissionDialog
-                        uuid={ bundleInfo.uuid }
-                        permission_spec={ bundleInfo.permission_spec }
-                        group_permissions={ bundleInfo.group_permissions }
-                        onChange={ this.props.onMetaDataChange }
-                    />
+                    <ConfigLabel label="Permissions:" inline={true}/>

```

Comment:
permission look is changed to be the same as header, added a icon to indicate editable